README.md

```diff
@@ -1,9 +1,65 @@
 # i5-white-label
-A plugin for Informer 5 that replaces the logo, app title, and favicon
+A plugin for Informer 5 that replaces various visual aspects of Informer
 
 
-## In order to use:
-1. Put the new app title into public/config.js
-2. Put the logo, banner logo (for login page) in to public/images
-3. Custom login page in assets/login.html
-4. To customize the favicon, use https://realfavicongenerator.net/ and put it in public/images/favicon folder
+## To replace Logo and Favicon
+1. Put the main logo and banner logo (for login page) into public/images
+2. Put custom login page in assets/login.html
+3. To customize the favicon, use https://realfavicongenerator.net/ and put it in public/images/favicon folder
+
+## To override various text and translations within the core Informer application
+Clients wanting to override (partially or wholly) any part of the core or plugin-added namespaces,
+or even want to introduce entirely new languages for their ENTIRE DEPLOYMENT can do so one of two ways:
+    
+1. For native environments, define a 'customLocalesDir' in config.json under i18n:
```

Comment:
property actually changed since this was initially written, so to separate -- `... define a custom 'localesDir' in ...`

Reply:
done